firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionGenerator.kt

```diff
@@ -46,6 +46,10 @@ internal class SessionGenerator(
   lateinit var currentSession: SessionDetails
     private set
 
+  /** Returns if a session has been generated. */
+  val hasGenerateSession: Boolean
```

Comment:
Where do we use this value?

Reply:
This is used as the condition to send the current session when a subscriber subscribes in `FirebaseSessions`.